test/addons/rook-cephfs/snap-class.yaml

```diff
@@ -0,0 +1,17 @@
+# SPDX-FileCopyrightText: The RamenDR authors
+# SPDX-License-Identifier: Apache-2.0
+
+# yamllint disable-line rule:line-length
+# Source: https://raw.githubusercontent.com/rook/rook/release-1.13/deploy/examples/csi/cephfs/snapshotclass.yaml
```

Comment:
Same reason as before, modeled around the rook-pool addon.

Same answer as before, source is provided for a diff in the future.

Are you insisting that what the delta is SHOULD be a comment in the YAML, or are you stating it should be used directly from Rook if not different? (I would need a common Rook version across the add-ons to do the latter)

(applies to all comments of this nature on sourced YAMLs)

Reply:
I'm thinking how we are going to update this when new rook version is released. We will need to diff the new rook config, and then understand the changes from our config vs the old rook config, and guess what was the indention of the author. Then think if we want to change something based on the changes in rook.

It will be easier it the yaml will document the intentions:
- Why we removed stuff
- Which values were changed, and why

In rook operator and cluster this is easy, we use kustomization and every change we made is documented.

If we don't change anything, using the origianl from rook is best, since we support caching kustomzizations.

We don't have global rook version for now, we duplicate the version in the kustomization yamls. I did not find a solution for keeping a version external to the kustomization, maybe you have an idea how to do this?